src/server/scripts/Spells/spell_paladin.cpp

```diff
@@ -965,6 +965,19 @@ class spell_pal_lay_on_hands : public SpellScript
         return true;
     }
 
+    void HandleHeal(SpellEffIndex /*effIndex*/)
```

Comment:
minor, but consider changing the handler's name to differentiate between a EffectHeal and EffectHealMaxHealth